### PR TITLE
Update profile.md

### LINKED ISSUE #104: Tidy up and provide more clarity to BLE profile page
https://github.com/microbit-foundation/dev-docs/blob/master/bluetooth/profile.md should have a clearer link to https://lancaster-university.github.io/microbit-docs/ble/profile/#all-services-enabled-hex-file and microbit samples where the Hex files live

### PR DESCRIPTION
**Pull request by microbit-mark**
http://review.review.tech.microbit.org.s3-website-eu-west-1.amazonaws.com/bluetooth/profile/

closes #104

### CODE REVIEW

File: bluetooth/profile.md [hunk 1]
```diff
@@ -3,54 +3,47 @@ layout: page
...
+The micro:bit supports Bluetooth via a single profile BBC micro:bit profile 
+which was custom developed for the device. This profile
+allows it to communicate with other BLE capable devices.
 
-If you are looking for information about the 'radio' feature, then
-that is not Bluetooth, it is a proprietary protocol from Nordic
-called Gazell, which you can find out more about in the links
-below.
+If you are looking for information about the 'radio' feature, it is a [proprietary protocol from Nordic
+called Gazell](https://developer.nordicsemi.com/nRF5_SDK/nRF51_SDK_v4.x.x/doc/html/group__gzll__02__user__guide.html). This is not Bluetooth.
```
Comment (jaustin) [inline bluetooth/profile.md]:
```suggestion
and Lancaster University](https://lancaster-university.github.io/microbit-docs/ubit/radio/). This is not Bluetooth.
```

Comment (microbit-mark) [reply bluetooth/profile.md]: Thanks @jaustin. Added the change

**Comment by martinwork** [comment]
The hex pointed to by "Full profile with the display unused" at the bottom of the page is DAL 1.4.17.

**Comment by microbit-mark** [comment]
This file is referenced in the dal docs under  https://github.com/lancaster-university/microbit-docs/blob/master/docs/ble/profile.md#gatt-services and located here https://github.com/lancaster-university/microbit-docs/blob/master/docs/resources/microbit-1_4_17_pwr0.zip does it need updating @microbit-sam ?

**Pull request merged**